go/vt/tabletserver/engines/schema/schema_engine.go

```diff
@@ -329,19 +329,6 @@ func (se *Engine) TableWasCreatedOrAltered(ctx context.Context, tableName string
 	return nil
 }
 
-// TableWasDropped must be called if a table was dropped.
-func (se *Engine) TableWasDropped(tableName sqlparser.TableIdent) {
```

Comment:
`TableWasCreatedOrAltered` is used by Reload. I can still remove from that function if you'd like. But I wasn't sure of the implications.